packages/modules/fsm/src/Module.js

```diff
@@ -41,17 +41,35 @@ import RateView from "./pages/citizen/Rating/RateView";
 const EmployeeApp = ({ path, url, userType }) => {
   const { t } = useTranslation();
   const location = useLocation();
-  const mobileView = innerWidth <= 640;
   const DSO = Digit.UserService.hasAccess(["FSM_DSO"]);
+  const [crumbs, setCrumbs] = useState([]);
+
+  useEffect(() => {
```

Comment:
useEffect can be avoided by using uselocation pathname and conditional rendering

Reply:
no bcz we have to update "show" whenever we go to any page. There is a way to avoid useEffect but then our app will no longer SPA